Jenkinsfile

```diff
@@ -16,6 +16,7 @@ pipeline {
     OPBEANS_REPO = 'opbeans-dotnet'
     BENCHMARK_SECRET  = 'secret/apm-team/ci/benchmark-cloud'
     SLACK_CHANNEL = '#apm-agent-dotnet'
+    AZURE_RESOURCE_GROUP_PREFIX = "ci-dotnet"
```

Comment:
This would need to include some unique identifier for a specific build run, so that the cleanup step only deletes resource groups created in this run. Is the Jenkins `BUILD_ID` unique per run? Could that be included?